integration-tests/src/test/java/oracle/weblogic/kubernetes/ItCrossDomainTransaction.java

```diff
@@ -360,13 +373,14 @@ private static void buildApplicationsAndDomains() {
    * If the server listen-addresses are resolvable between the transaction
    * participants, then the transaction should complete successfully
    */
+  @Order(1)
   @Test
   @DisplayName("Check cross domain transaction works")
   void testCrossDomainTransaction() {
 
     String curlRequest = String.format("curl -v --show-error --noproxy '*' "
-            + "http://%s/TxForward/TxForward?urls=t3://%s.%s:7001,t3://%s1.%s:8001,t3://%s1.%s:8001,t3://%s2.%s:8001",
-        hostAndPort, domain1AdminServerPodName, domain1Namespace,
+            + "http://%s:%s/TxForward/TxForward?urls=t3://%s.%s:7001,t3://%s1.%s:8001,t3://%s1.%s:8001,t3://%s2.%s:8001",
```

Comment:
I believe this was added to support this test in OKD, why are we rolling it back?

Reply:
and similar changes below.

Reply:
sync from the main branch